Classes/Domain/Index/AbstractIndexer.php

```diff
@@ -30,6 +32,16 @@
     protected $connection;
 
     /**
+     * @var ConfigurationContainerInterface
+     */
+    protected $configuration;
+
+    /**
+     * @var string
+     */
+    protected $identifier;
```

Comment:
In other parts of code you initialized it with an empty string.
Shouldn't this be consistent?